js/plugins/citationFormats.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file js/plugins/citationFormats.js
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @brief Frontend script to fetch citation formats on demand.
+ *
+ * This script is designed to be compatible with any themes that follow these
+ * steps. First, you need a target element that will display the citation in
+ * the requested format. This should have an id of `citationOutput`:
+ *
+ * <div id="citationOutput"></div>
+ *
+ * You can create a link to retrieve a citation and display it in this div by
+ * assigning the link a `data-load-citation` attribute:
+ *
+ * <a href="{url ...}" data-load-citation="true">View citation in ABNT format</a>
+ *
+ * Downloadable citations should leave the `data-load-citation` attribute out
+ * to allow normal browser download handling.
+ *
+ * This script requires jQuery. The format you specify should match
+ * a format provided by a CitationFormat plugin.
+ */
+(function($) {
+
+	// Require jQuery
+	if (typeof $ === 'undefined') {
+		console.log('The Citation Formats handler could not be loaded because jQuery was not found.');
```

Comment:
Thanks, @NateWr, I also lean towards using JSHint instead of JSLint. I suggest tackling that post-3.0, however. In the interim, this script would be neither linted or minified. Are you content with that for the moment?